magda-web-client/src/Components/Editing/Editors/multiItem.tsx

```diff
@@ -137,7 +149,8 @@ export class ListMultiItemEditor<V> extends MultiItemEditor<V> {
         singleEditor: Editor<V>,
         createNewValue: () => V,
         canBeAdded: (value: V) => boolean = value => !!value,
-        addOnChange: boolean = false
+        addOnChange: boolean = false,
+        renderAbove: boolean = true
```

Comment:
It's this one! :)

Reply:
This is setting a default value of `true`. But since it has a default value, the `!!` is indeed unnecessary since `renderAbove` should always have a concrete value.